src/handlers.rs

```diff
@@ -27,45 +24,51 @@ pub async fn ingest(
 ) -> impl IntoResponse {
     debug!("received payload");
 
-    let signature = match headers.get("x-vercel-signature") {
-        Some(signature) => signature.to_str().unwrap(),
-        None => {
-            warn!("received payload without signature");
-            counter!("drain_recv_invalid_signature").increment(1);
-            // Vercel's verificaion request doesn't actually sign the request,
-            // even if you set a custom secret in advance.
-            return Response::builder()
-                .status(StatusCode::OK)
-                .body(Body::empty())
-                .expect("Defined Responses to be infalliable.");
-        }
+    let Some(sig_header) = headers.get("x-vercel-signature") else {
+        // Vercel's verification request isn't signed, even if you set a custom
+        // secret in advance. It also expect 200 OK.
+        //
+        // Configure Vercel to send a random custom static header and
+        // configure your HTTPS load balancer in front of vercel-log-drain
+        // to require it, if you want to discard bot traffic.
```

Comment:
```suggestion
```

let's just remove this comment since it is documented in the recommended setup process.